src/detect-flowbits.c

```diff
@@ -69,6 +72,49 @@ void DetectFlowbitsRegister (void)
     DetectSetupParseRegexes(PARSE_REGEX, &parse_regex);
 }
 
+static int FlowbitOrAddData(DetectEngineCtx *de_ctx, DetectFlowbitsData *cd, char *arrptr)
+{
+    char *strarr[MAX_TOKENS];
+    char *token;
+    char *saveptr = NULL;
+    uint8_t i = 0;
+
+    while ((token = strtok_r(arrptr, "|", &saveptr))) {
+        // Check for leading/trailing spaces in the token
+        while(isspace((unsigned char)*token)) token++;
+        if (*token == 0)
+            continue;
+        char *end = token + strlen(token) - 1;
```

Comment:
Why the `-1`?

Reply:
From what I understood, if we have the following string (`.` represents space char) with char addresses written below each char,
```
a b c . . .
1 2 3 4 5 6
^
token
```
then `token + strlen(token) = 1 + 6 = 7` but the last character is at `6` hence the `-1`.
Is there something wrong?